src/test/java/StringManipulationTest.java

```diff
@@ -27,18 +29,23 @@ public void testCount1() {
     }
 
     @Test
+    // test when no words are present
     public void testCount2() {
-        fail("Not yet implemented");
+        manipulatedstring.setString("n0 w0rds");
+        assertEquals(0, manipulatedstring.count());
     }
 
     @Test
```

Comment:
Based on the comment I made in the StringManipulation.java, I would suggest to edit this test so it counts 4 words instead of 2.